From 2bd261c0716362b50614e4e9523b8567a95ef3c2 Mon Sep 17 00:00:00 2001
From: Lu Yahan <yahan@iscas.ac.cn>
Date: Tue, 17 May 2022 09:18:56 +0800
Subject: [PATCH] [riscv64][wasm-relaxed-simd] Keep q15_mul emit operations
 consistent

Port commit d2a8bdec7fb749ab0cd44bb29cb4aad0b9dc0a38
Bug: v8:12284

Change-Id: I8ec0c2270d85d8068823e241e1c326aa90caad8c
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3651507
Auto-Submit: Yahan Lu <yahan@iscas.ac.cn>
Commit-Queue: Yahan Lu <yahan@iscas.ac.cn>
Commit-Queue: ji qiu <qiuji@iscas.ac.cn>
Reviewed-by: ji qiu <qiuji@iscas.ac.cn>
Cr-Commit-Position: refs/heads/main@{#80573}
---
 src/wasm/baseline/riscv64/liftoff-assembler-riscv64.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/wasm/baseline/riscv64/liftoff-assembler-riscv64.h b/src/wasm/baseline/riscv64/liftoff-assembler-riscv64.h
index 86e878ff0f2..c9c24c2e02b 100644
--- a/src/wasm/baseline/riscv64/liftoff-assembler-riscv64.h
+++ b/src/wasm/baseline/riscv64/liftoff-assembler-riscv64.h
@@ -2167,7 +2167,7 @@ void LiftoffAssembler::emit_i16x8_q15mulr_sat_s(LiftoffRegister dst,
   vsmul_vv(dst.fp().toV(), src1.fp().toV(), src2.fp().toV());
 }
 
-void LiftoffAssembler::emit_relaxed_i16x8_q15mulr_s(LiftoffRegister dst,
+void LiftoffAssembler::emit_i16x8_relaxed_q15mulr_s(LiftoffRegister dst,
                                                     LiftoffRegister src1,
                                                     LiftoffRegister src2) {
   VU.set(kScratchReg, E16, m1);
-- 
2.35.1

